Makefile

```diff
@@ -1,8 +1,85 @@
 dev_build_version=$(shell git describe --tags --always --dirty)
 
+# TODO: run golint and errcheck, but only to catch *new* violations and
+# decide whether to change code or not (e.g. we need to be able to whitelist
+# violations already in the code). They can be useful to catch errors, but
+# they are just too noisy to be a requirement for a CI -- we don't even *want*
+# to fix some of the things they consider to be violations.
+.PHONY: ci
+ci: deps checkgofmt vet staticcheck ineffassign predeclared test
+
+.PHONY: bigtable
+bigtable:
```

Comment:
I thought about having to use an env variable to specify in which project we will run the target.

We could do for example make bigtable ci, or make storage ci. We could also add another target that runs both, but the main idea was to not need to change a lot of things if we add another emulator